deployment/ansible/roles/model-my-watershed.app/templates/nginx-app.conf.j2

```diff
@@ -10,6 +10,10 @@ server {
   location /version.txt {
     alias /srv/version.txt;
   }
+
+  location /favicon.ico {
+    alias {{ app_static_root }}favicon.png;
+  }
```

Comment:
Makes sense. No good reason to guard it. Fixed in https://github.com/WikiWatershed/model-my-watershed/commit/c52ed908666ad2443d4c5ff7c4740a9bed3f23df.